tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "exclude": ["integrationTests/ts/**/*"],
   "compilerOptions": {
     "module": "commonjs",
-    "lib": ["es2019", "es2020.promise", "es2020.bigint", "es2020.string"],
+    "lib": ["es2019", "es2020.string"],
```

Comment:
No, keep this as is.
Not all code published as an NPM package and nothing preventing us from using these types there.